api/src/models/nfl-teams.model.ts

```diff
@@ -0,0 +1,62 @@
+import { Sequelize, DataTypes, Model, Optional } from 'sequelize';
+import { NFLTeam } from '@/interfaces/nfl-teams.interface';
+
+export type NFLTeamCreationAttributes = Optional<NFLTeam, 'id' | 'team_name_short' 
+  | 'team_name_long' | 'team_city_long' | 'team_city_short' | 'team_logo'>;
+
+export class NFLTeamModel extends Model<NFLTeam, NFLTeamCreationAttributes> implements NFLTeam {
+  public id: number; 
+  public team_name_short: string; 
+  public team_name_long: string; 
+  public team_city_short: string; 
+  public team_city_long: string; 
+  public team_logo: string; 
+
+  public readonly createdAt!: Date;
+  public readonly updatedAt!: Date;
+}
+  
+export default function (sequelize: Sequelize): typeof NFLTeamModel {
+  NFLTeamModel.init(
```

Comment:
Does this essentially create the NFLTeamModel table in the database?

Reply:
Yes, I think so. This model class represents the `NFLTeams` table in the database (you can see the field `tableName` where we specify the table name). If you create an instance of this class, then that would be a single row in the SQL table. I think that the fields in this class represent the specific columns in the table (id, team name short, etc.). 

Here is the basic docs on the sequelize models: https://sequelize.org/docs/v6/core-concepts/model-basics/
And here is more in depth docs on models that explains it: https://sequelize.org/api/v6/class/src/model.js~model

Reply:
Thanks